scripts/route-matchers.ts

```
@@ -0,0 +1,572 @@
+import { Effect, Console, Array, String, pipe, Data } from "effect";
```

Biome (1.9.4)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)